src/NodaTime/Text/TypeConverterSettings.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2020 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NodaTime.Utility;
+
+namespace NodaTime.Text
+{
+    /// <summary>
+    /// Ambient settings applied for the default type converters. There's no simple context
+    /// to express this information in a more elegant way, although users can create their
+    /// own type converters to apply if they wish to.
+    /// </summary>
+    public static class TypeConverterSettings
+    {
+        private static readonly object stateLock = new object();
+
+        private static IDateTimeZoneProvider dateTimeZoneProvider = DateTimeZoneProviders.Tzdb;
+
+        /// <summary>
+        /// Gets the <see cref="IDateTimeZoneProvider"/> to use to interpret a time zone ID read as part of
```

Comment:
It's also used for serialisation, not just deserialisation. Not sure if we should be clearer about that here?

(Same goes for `XmlSerializationSettings`.)

Reply:
I don't think it really is used for serialization - there's no need to look anything up. We just serialize the ID of the zone, regardless of which provider it came from.

Reply:
Ah, that's a good point. Thanks!